chromium/background.js

```diff
@@ -674,5 +683,57 @@ chrome.runtime.onMessage.addListener(function(message, sender, sendResponse){
     updateState();
   } else if (message.type == "remove_rule") {
     removeRule(message.object);
+  } else if (message.type == "import_settings") {
```

Comment:
Can you add a comment explaining when this is used? I.e., importing from the Options page rather than automatically from the parent extension?